README.md

```diff
@@ -389,12 +389,19 @@ Board optional configurations.
 * `simulatorFirmwareVersion` {String} - Allows the simulator to use firmware version 2 features. (2 Possible Options)
   * `v1` - Firmware Version 1 (Default)
   * `v2` - Firmware Version 2
+* `simulatorFragmentation` {String} - Specifies how to break packets to simulate fragmentation. (4 Possible Options)
```

Comment:
Default should be `none` and if this turns out to be very useful then we can propose to make `random` the default in 2.x.x because it's a major change to how the simulator works IMO.